react/src/components/organisms/HelpTip/index.js

```diff
@@ -67,68 +73,72 @@ class HelpTip extends Component {
+    const childArray = children && (Array.isArray(children) ? children : [children]);
 
     return(
       <span className={baseClass} id={id}>
-        <span className="ma__help-tip__label" id={labelId}>
-          {this.buildDangerouslyIfHasMarkup(this.props.textBefore, hasMarkup)}
-          {/* can't use a button b/c disrupts fieldsets when used inside of legend */}
-          <div
-            className={`ma__help-tip__trigger ${this.state.isOpen ? 'active' : ''}`}
-            onClick={this.toggleOpen}
-            onKeyUp={this.toggleOpenForKeyUp}
-            tabIndex="0"
-            role="button"
-            aria-describedby={id}
-            aria-expanded={this.state.isOpen}
-          >
-            {this.buildDangerouslyIfHasMarkup(triggerText, hasMarkup)}
-            <Icon name="questionmark" svgHeight={15} svgWidth={15} />
-          </div>
-          {this.buildDangerouslyIfHasMarkup(textAfter, hasMarkup)}
-        </span>
-        <Collapse in={this.state.isOpen} dimension="height" className={helpTextContainer}>
-          <div className="ma__help-tip__content">
-            <div
+        {triggerText.map((trigger, index) => (
+          <span key={`HelpTip-${labelId}-${index}`} className="ma__help-tip__label" id={`${labelId}-${index}`}>
+            {index === 0 && this.buildDangerouslyIfHasMarkup(splitText[index], hasMarkup)}
+            <span
+              className={`ma__help-tip__trigger ${this.state.isOpen[index] ? 'active' : ''}`}
+              onClick={() => this.toggleOpen(index)}
+              onKeyUp={(e) => this.toggleOpenForKeyUp(e, index)}
               tabIndex="0"
               role="button"
-              className="ma__help-tip__close-mobile"
-              onClick={this.toggleOpen}
-              onKeyUp={this.toggleOpenForKeyUp}
+              aria-describedby={id}
+              aria-expanded={this.state.isOpen[index]}
             >
-              <Icon name="close" label="Close help tip" />
-            </div>
-            <div
-              tabIndex="0"
-              role="button"
-              className="ma__help-tip__close-desktop"
-              onClick={this.toggleOpen}
-              onKeyUp={this.toggleOpenForKeyUp}
-            >
-              <Icon name="close" label="Close help tip" />
-            </div>
-            {(helpText || children) && (
-              <div className={helpTextClasses} aria-live="polite">
-                {children || (<p className={helpTextDirect}>{helpText}</p>)}
+              {this.buildDangerouslyIfHasMarkup(trigger, hasMarkup)}
+              <Icon name="questionmark" svgHeight={15} svgWidth={15} />
+            </span>
+            {this.buildDangerouslyIfHasMarkup(splitText[index + 1], hasMarkup)}
+          </span>
+        ))}
+        {triggerText.map((trigger, index) => (
+          <Collapse key={`HelpTip-collapse-${labelId}-${index}`} in={this.state.isOpen[index]} dimension="height" className={helpTextContainer}>
```

Comment:
Can we use id here instead of labelId, so we don't need to add this required prop?